### PR TITLE
VZ-8104 OAM Operator pod security

### PR DESCRIPTION
**Pull request by mbenmclean**
This change adds pod security for the OAM operator. This required modifying the oam operator image, so this change also picks up the new image.

### CODE REVIEW

File: platform-operator/helm_config/overrides/oam-kubernetes-runtime-values.yaml [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
 
 # Removing the limits for OAM since there could be many OAM apps deployed
 resources:
   requests:
     cpu: 100m
-    memory: 20Mi
\ No newline at end of file
+    memory: 20Mi
+
+podSecurityContext:
```
Comment (michael-cico) [inline platform-operator/helm_config/overrides/oam-kubernetes-runtime-values.yaml:10]: do we need to use the user/group IDs?

Comment (mbenmclean) [reply platform-operator/helm_config/overrides/oam-kubernetes-runtime-values.yaml:10]: It sets it correctly by default. I can add it explicitly if you'd like.

Comment (mbenmclean) [reply platform-operator/helm_config/overrides/oam-kubernetes-runtime-values.yaml:10]: Actually it doesn't I'll update that.

File: platform-operator/helm_config/overrides/oam-kubernetes-runtime-values.yaml [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+  fsGroup: 65532
```
Comment (pfmackin) [inline platform-operator/helm_config/overrides/oam-kubernetes-runtime-values.yaml:11]: is fsGroup really needed?

**Review by michael-cico — APPROVED** [review]

**Pull request merged**